dGame/dUtilities/Mail.h

```diff
@@ -1,43 +1,217 @@
-#pragma once
+#ifndef __MAIL_H__
```

Comment:
```suggestion
#ifndef MAIL_H
```
Leading underscore defines are reserved

Reply:
This is not consistent across the codebase, that should so be addressed in a separate PR if we want to enforce that

Reply:
Fair enough